.github/workflows/pull_request.yml

```diff
@@ -50,6 +50,360 @@ jobs:
       with:
         image: "openfoodfacts-server/backend:dev"
 
+  check_perl_formatting:
```

Comment:
It's cool to have written this check.

That said it may be better if all this is part of a Make command and it uses docker, so that we can check it locally and also lint, because if not debugging will be hard !

So we need to add perltidy to cpanfile (in dev section). and add the right targets to Makefile.